dev/automated_tests/flutter_test/print_user_created_ancestor_expectation.txt

```diff
@@ -0,0 +1,27 @@
+<<skip until matching line>>
+══╡ EXCEPTION CAUGHT BY WIDGETS LIBRARY ╞═══════════════════════════════════════════════════════════
+The following assertion was thrown building
+RawGestureDetector-\[LabeledGlobalKey<RawGestureDetectorState>#.+\]\(state:
+RawGestureDetectorState#.+\(gestures: <none>, behavior: opaque\)\):
+'package:flutter\/src\/painting\/basic_types\.dart': Failed assertion: line 223 pos 10: 'textDirection
+!= null': is not true\.
+
+Either the assertion indicates an error in the framework itself, or we should provide substantially
+more information in this error message to help you determine and fix the underlying cause\.
+In either case, please report this assertion by filing a bug on GitHub:
+  https:\/\/github\.com\/flutter\/flutter\/issues\/new\?template=BUG\.md
+
+User created ancestor of the error causing widget was:
+
```

Comment:
This formatting needs to be made consistent with other error message formatting.
Other messages do not add a whitespace line after a colon and typically code following lines with a colon is indented 2 spaces to indicate the parent child relationship.
So if you want to keep the Widget class on its own line I would recommend the output
```
User created ancestor of the error causing widget was:
  CustomScrollView
  file:\/\/\/.+print_user_created_ancestor_test\.dart:[0-9]+:7 

When the exception was thrown, this was the stack:
<<skip until matching line>>
```
I'll comment later in the CL on how to generate that output.

Reply:
Fyi @InMatrix for feedback on whether this display is what we want.